### PR TITLE
Small improvment login page

### PR DESCRIPTION
**Pull request by Lainow**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #15258

Improve login page

**Before**
image

**After**
image

### CODE REVIEW

**Review by AdrienClairembault — CHANGES_REQUESTED** [review]
Login text and FAQ button are not centered, can you fix it ?

image

**Review by AdrienClairembault — APPROVED** [review]

File: templates/pages/login.html.twig [hunk 1]
```diff
@@ -34,97 +34,101 @@
...
-                     <i class="fas fa-question"></i>&nbsp;
-                     {{ __('FAQ') }}
-                  </a>
-               {% endif %}
+				{% if errors|length > 0 %}
+					<hr/>
+					<div class="alert alert-danger" role="alert">
+						{{ errors }}
+					</div>
+				{% endif %}
```
Comment (cedric-anne) [inline templates/pages/login.html.twig:104]: `errors` variable is no longer defined here, as errors rendering has be moved to the `login_error.html.twig` template, see #16289.

**Review by cedric-anne — CHANGES_REQUESTED** [review]

**Review by cedric-anne — COMMENTED** [review]
Seems OK, but whitespace changes (spaces -> tabs) should be reverted.

**Review by cedric-anne — APPROVED** [review]

File: templates/pages/login.html.twig [hunk 2]
```diff
@@ -34,97 +34,94 @@
...
-               {{ call_plugin_hook(constant('Glpi\\Plugin\\Hooks::DISPLAY_LOGIN')) }}
+            {% if right_panel %}
+                <div class="col-auto px-2 text-center">
+                    {{ call_plugin_hook(constant('Glpi\\Plugin\\Hooks::DISPLAY_LOGIN')) }}
+                </div>
+            {% endif %}
+        </div>
+        {% if config('use_public_faq') %}
+            <div class="text-center mt-4 border-top">
+                <a class="btn btn-outline-secondary mt-4" href="{{ path(front/helpdesk.faq.php) }}">
```
Comment (AdrienClairembault) [inline templates/pages/login.html.twig:108]: Shouldn't the supplied path be a string ?

Comment (cedric-anne) [reply templates/pages/login.html.twig]: Fixed.

**Review by AdrienClairembault — APPROVED** [review]

**Review by orthagh — APPROVED** [review]

**Pull request merged**